tests/integration/java/com/github/bazel_contrib/target_determinator/integration/BazelDifferIntegrationTest.java

```diff
@@ -53,7 +56,19 @@ private static void runProcess(Path workingDirectory, String... argv)
     // Do not clean the environment so we can inherit variables passed e.g. via --test_env.
     // Useful for CC (needed by bazel).
     processBuilder.environment().put("HOME", System.getProperty("user.home"));
-    processBuilder.environment().put("PATH", System.getenv("PATH"));
+
+    // There is a bug in `bazel-differ` where in `internal/bazel.go` they default to always using `bazel`
```

Comment:
would it not be easier to put the path to "our" bazel as the first element in `PATH` instead of filtering it like this?

Reply:
We don't really know which is "our" bazel at this point.